flink-ml-lib/pom.xml

```diff
@@ -106,6 +106,11 @@ under the License.
       <type>jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I think this dependency is unnecessary here, given that we move all infra to `flink-ml-core`.